dataland-frontend/src/components/forms/parts/elements/basic/CheckboxesListFormElement.vue

```diff
@@ -60,7 +60,8 @@ export default defineComponent({
         this.currentValue = checkboxValue[0].toString();
         this.$emit("update:currentValue", checkboxValue[0].toString());
       } else {
-        this.shouldBeIgnored = true;
+        this.currentValue = null;
+        this.shouldBeIgnored = !this.validation.includes("is:");
         this.$emit("update:currentValue", null);
```

Comment:
Why do we need this emit?
I have searched all our frontend code and found out that only "YesNoFormField.vue" uses this emit.
And it simply emits it again to its parent.
But nowhere in the code I found a parent of a YesNoFormField that actually does something with this event.
So it seems to me that we can remove it.

Reply:
I removed it and everything still seems to work.
Can you review this change pls on Monday?

Reply:
Yes you are right. It seems that nothing is listening to this event

Reply:
Ok nice then I am resolving this comment